libcontainer/specconv/spec_linux.go

```diff
@@ -616,6 +616,15 @@ func CreateCgroupConfig(opts *CreateOpts, defaultDevs []*devices.Device) (*confi
 					Limit:    l.Limit,
 				})
 			}
+			if len(r.Rdma) > 0 {
+				for k, v := range r.Rdma {
+					c.Resources.Rdma = make(map[string]configs.LinuxRdma, len(r.Rdma))
```

Comment:
This is plain wrong -- you recreate a whole map for every iteration. This should be outside of the loop.

Reply:
resolved.